src/lager_udp_backend.erl

```diff
@@ -78,7 +73,7 @@ init(Params)->
         port=Port,
         socket=Socket,
         formatter=Formatter,
-        format_config=FormatConfig}
+        formatter_config=FormatterConfig}
```

Comment:
And spaces around `=` are also worth adding.

Reply:
The same comment as above applies here 🙂